packaging/tags.py

```diff
@@ -57,6 +58,24 @@
 _32_BIT_INTERPRETER = sys.maxsize <= 2 ** 32
 
 
+_LEGACY_MANYLINUX_MAP = {
+    # CentOS 7 w/ glibc 2.17 (PEP 599)
+    (2, 17): "manylinux2014",
+    # CentOS 6 w/ glibc 2.12 (PEP 571)
+    (2, 12): "manylinux2010",
+    # CentOS 5 w/ glibc 2.5 (PEP 513)
+    (2, 5): "manylinux1",
+}
+
+# If glibc ever changes its major version, we need to know what the last
+# minor version was, so we can build the complete list of all versions.
+# For now, guess what the highest minor version might be, assume it will
+# be 50 for testing. Once this actually happens, update the dictionary
+# with the actual value.
+_LAST_GLIBC_MINOR = collections.defaultdict(lambda: 50)  # type: Dict[int, int]
+version = collections.namedtuple("Version", ["major", "minor"])
```

Comment:
```suggestion
glibcVersion = collections.namedtuple("glibcVersion", ["major", "minor"])
```

To avoid any confusion with `packaging.version`: https://packaging.pypa.io/en/latest/version/.

Reply:
ok